Detections/MultipleDataSources/RiskyUserIn3Pnetworkactivity.yaml

```diff
@@ -0,0 +1,91 @@
+id: 042f2801-a375-4cfd-bd29-041fc7ed88a0
+name: Risky User observed in non-microsoft network device 
+description: |
+  'This content is utilized to identify instances of successful login by risky users, who have been observed engaging in potentially suspicious network activity on non-Microsoft network devices.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId:  AzureActiveDirectory
+    dataTypes:
+        - SigninLogs 
+  - connectorId:  PaloAltoNetworks
+    dataTypes:
+    - CommonSecurityLog (PaloAlto)
+  - connectorId:  Fortinet
+    dataTypes:
+    - CommonSecurityLog (Fortinet)
+  - connectorId:  CheckPoint
+    dataTypes:
+    - CommonSecurityLog (CheckPoint)
+  - connectorId:  Zscaler
+    dataTypes:
+    - CommonSecurityLog (Zscaler)
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1071
+query: |
+
+    SigninLogs
+    //Find risky Signin
+    | where RiskState == "atRisk" and ResultType == 0
+    | extend Signin_Time = TimeGenerated
+    | summarize
+        AppDisplayName=make_set(AppDisplayName),
+        ClientAppUsed=make_set(ClientAppUsed),
+        UserAgent=make_set(UserAgent),
+        CorrelationId=make_set(CorrelationId),
+        Signin_Time= min(Signin_Time),
+        RiskEventTypes=make_set(RiskEventTypes)
+        by
+        ConditionalAccessStatus,
+        IPAddress,
+        IsRisky,
+        ResourceDisplayName,
+        RiskDetail,
+        ResultType,
+        RiskLevelAggregated,
+        RiskLevelDuringSignIn,
+        RiskState,
+        UserPrincipalName=tostring(tolower(UserPrincipalName)),
+        SourceSystem
+    | join kind=inner (
+        CommonSecurityLog
+        | where DeviceVendor has_any  ("Palo Alto Networks", "Fortinet", "Check Point", "Zscaler")
+        | where DeviceAction != "Block"
```

Comment:
Is just 'Block' enough or are there other Actions we see that we need to include?

Reply:
Only the block action is stopping network activity; the other redirect and allowed and other actions are successful network actions inside our scope.